packages/files/file.pony

```diff
@@ -1,4 +1,21 @@
-primitive _FileHandle
+use @_read[I32](fd: I32, buffer: Pointer[None], bytes_to_read: I32) if windows
+use @read[ISize](fd: I32, buffer: Pointer[None], bytes_to_read: USize) if not windows
+use @_write[I32](fd: I32, buffer: Pointer[None], bytes_to_send: I32) if windows
+use @writev[ISize](fd: I32, buffer: Pointer[None], num_to_send: I32) if not windows
+use @_lseeki64[I64](fd: I32, offset: I64, base: I32) if windows
+use @lseek[I64](fd: I32, offset: I64, base: I32) if not windows
```

Comment:
The definition of `lseek` on Linux is quite complex:

```C
#ifndef __USE_FILE_OFFSET64
extern __off_t lseek (int __fd, __off_t __offset, int __whence) __THROW;
#else
# ifdef __REDIRECT_NTH
extern __off64_t __REDIRECT_NTH (lseek,
				 (int __fd, __off64_t __offset, int __whence),
				 lseek64);
# else
#  define lseek lseek64
# endif
#endif
```

What this means is that `lseek` is actually, at a binary level, the old 32-bit function. Because `ponyc` doesn't examine C header files, it can't know about `__REDIRECT_NTH` (which is a macro that uses inline assembly to create a function alias), so I believe it will call the 32-bit `lseek` here.

Or I'm wrong and it's fine as it is :)

Any way we can double check this?